app/src/main/res/layout/search_selector.xml

```diff
@@ -30,7 +29,6 @@
                 android:id="@+id/icon"
                 android:layout_width="24dp"
                 android:layout_height="24dp"
-                android:scaleType="center"
```

Comment:
Trying to understand this change. Could you explain what kind of image scale issues you were seeing here and what was the mitigation?

Reply:
I tried to explain it [here](https://github.com/mozilla-mobile/fenix/pull/27647#issuecomment-1301656042).

In short, this scaling type would create extra margin for icons bigger then the view size. It can be seen with duck duck go icon, if one revers the change. Default scaling already works correctly for [searchIcon](https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/layout/fragment_home.xml#138), so I took that approach

Reply:
here is the [patch](https://github.com/mozilla-mobile/fenix/pull/27726) with reverted scaling code. with unified search is on, duckduck, amazon, google icons will show the problem! one could also play around with `preference_icon_drawable_size` parameter to see if it has any effect.